backend-opal/src/server/watchLists.js

```diff
@@ -10,6 +10,12 @@ const data = [
   }
 ];
 
+function checkRequestBody(body) {
+  const bodyKey = ['id', 'userId', 'name', 'symbols'];
```

Comment:
@Yuan-EPAM `id` and `symbols` won't be part of the request
- id should be generated
- symbols is empty by default, we can add to them later